lp-simulation-environment/monitoring/src/main/java/eu/learnpad/simulator/api/impl/SimulatorMonCoreFacadeRESTResource.java

```diff
@@ -0,0 +1,185 @@
+package eu.learnpad.simulator.api.impl;
+
+import java.io.IOException;
+import java.util.List;
+
+import org.apache.commons.httpclient.HttpClient;
+import org.apache.commons.httpclient.methods.PostMethod;
+import org.apache.commons.net.ntp.TimeStamp;
+
+import eu.learnpad.exception.LpRestException;
+import eu.learnpad.sim.rest.data.UserData;
+import eu.learnpad.sim.rest.event.impl.ProcessEndEvent;
+import eu.learnpad.sim.rest.event.impl.ProcessStartEvent;
+import eu.learnpad.sim.rest.event.impl.SessionScoreUpdateEvent;
+import eu.learnpad.sim.rest.event.impl.SimulationEndEvent;
+import eu.learnpad.sim.rest.event.impl.SimulationStartEvent;
+import eu.learnpad.sim.rest.event.impl.TaskEndEvent;
+import eu.learnpad.sim.rest.event.impl.TaskFailedEvent;
+import eu.learnpad.sim.rest.event.impl.TaskStartEvent;
+import eu.learnpad.simulator.mon.utils.DebugMessages;
+import eu.learnpad.simulator.api.impl.utils.RestResource;;
+
+public class SimulatorMonCoreFacadeRESTResource implements eu.learnpad.sim.CoreFacade {
+
+	@Override
+	public List<String> getUsers() {
+		// TODO Auto-generated method stub
+		return null;
+	}
+
+	@Override
+	public UserData getUserData(String userartifactid) {
+		return null;
+	}
+
+	@Override
+	public void receiveSimulationStartEvent(SimulationStartEvent event) throws LpRestException {
+		HttpClient httpClient = RestResource.getClient();
+		String uri = String.format("%s/simulationstart/%s",
```

Comment:
All the URL should be revised in `lp-cp-apis` (look [here](https://github.com/LearnPAd/learnpad/blob/master/lp-core-platform/lp-cp-apis/src/main/java/eu/learnpad/sim/rest/IEventReceiver.java)).